config/routes.json

```diff
@@ -1,3 +1,3 @@
 {
-    "/apps/host-inventory*": { "url": "http://host.docker.internal:8003" }
+    "/apps/inventory*": { "url": "http://host.docker.internal:8003" }
```

Comment:
What is the reason for this change? Since there are many Inventories in Insights, I would prefer if we kept the name `host-inventory`. Even better would be `insights-host-inventory`.

Reply:
This is the app name for inventory, and it was causing the issue for the setup part, so we need to fix it